### PR TITLE
added call to action to homepage

### PR DESCRIPTION
**Pull request by danielfmiranda**
# Description

Related PRs/issues: #9357
Screenshot from Figma:
image

This PR adds the buyersguide call to the action to the home page, aligned with the other row items according to the figma file.

# Screenshots
Desktop:
Screenshot 2022-09-20 at 13-09-11 Privacy Not Included A Buyer’s Guide for Connected Products

Mobile:
Screen Shot 2022-08-23 at 6 33 05 PM

# Checklist

**Tests**
- [ ] ~~Is the code I'm adding covered by tests?~~

**Changes in Models:**
- [ ] ~~Did I update or add new fake data?~~
- [ ] ~~Did I squash my migration?~~
- [ ] ~~Are my changes [backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?~~

**Documentation:**
- [ ] ~~Is my code documented?~~
- [ ] ~~Did I update the READMEs or wagtail documentation?~~

### CODE REVIEW

File: network-api/networkapi/templates/pages/buyersguide/home.html [hunk 1]
```diff
@@ -21,31 +21,36 @@
...
+            {% if product_updates %}
+              <div class="{{row_item_classes}}">
+                {% include "fragments/buyersguide/in_the_press.html" %}
+              </div>
+            {% endif %}
+          {% endwith %}
+
+          {% with cta=featured_cta %}
+            {% if cta %}
+              <div class="{{row_item_classes}} large:-tw-mt-[20px]">
```
Comment (danielfmiranda) [inline network-api/networkapi/templates/pages/buyersguide/home.html:44]: this negative margin makes sure that the callout box is top-lined with the rest of the items, as shown in the figma comp!

**Review by fessehaye — CHANGES_REQUESTED** [review]
Alignment is off for desktop, I think removing the negative margin would be more accurate

**Comment by danielfmiranda** [comment]
Thanks @fessehaye! Updated

**Review by fessehaye — APPROVED** [review]
thanks daniel!

**Pull request merged**